pylab/accounts/tests/test_settings.py

```diff
@@ -8,7 +8,9 @@
 class SettingsTests(django_webtest.WebTest):
     def setUp(self):
         super().setUp()
-        auth_models.User.objects.create_user('u1')
+        u1 = auth_models.User.objects.create_user('u1')
+        u1.profile.accepted_terms = True
+        u1.profile.save()
```

Comment:
Instead, you could use factory boy. Factory for user is already defined in `pylab.website.core.factories`. Since `accepted_terms` should be `True` in most tests, then I guess it should be added to factory as default value.